kie-server-parent/kie-server-services/kie-server-services-common/pom.xml

```diff
@@ -206,7 +206,24 @@
       <groupId>org.kie.soup</groupId>
       <artifactId>kie-soup-maven-support</artifactId>
     </dependency>
-
+    <!--<dependency>-->
+      <!--<groupId>org.keycloak</groupId>-->
+      <!--<artifactId>keycloak-core</artifactId>-->
+      <!--<exclusions>-->
+        <!--<exclusion>-->
+          <!--<groupId>org.bouncycastle</groupId>-->
+          <!--<artifactId>bcprov-jdk15on</artifactId>-->
+        <!--</exclusion>-->
+        <!--<exclusion>-->
+          <!--<groupId>org.bouncycastle</groupId>-->
+          <!--<artifactId>bcpkix-jdk15on</artifactId>-->
+        <!--</exclusion>-->
+      <!--</exclusions>-->
+    <!--</dependency>-->
+    <dependency>
+      <groupId>org.wildfly.security</groupId>
+      <artifactId>wildfly-elytron</artifactId>
```

Comment:
wouldn't this cause some issues in WAS and WLS? has this been tested on the other containers? I am against adding application server specific code/dependencies into kie server

Reply:
I think you are right, the dependency just for wildfly elyton. I will try to find some way add specific code/dependencies .